submissions/Oleksandr-Heleta/popup/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="./popup-icons/favicon.png" type="image/x-icon">
+    <title>popup</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <ul class="nav">
+                <li class="nav_item"><a href="#" target="_blank" rel="noopener noreferrer">Mail</a></li>
+                <li class="nav_item"><a href="#" target="_blank" rel="noopener noreferrer">Image</a></li>
+                <li class="nav_item">
+                    <a href="#" target="_blank">
+                        <img class="nav-img" src="./popup-icons/bell.png" alt="bell">
+                    </a>
+                </li>
+                <li class="popup_button nav_item">
+
+                    <input class="popup_button-input" type="checkbox" name="menu" id="menu">
+                    <label class="popup_button-label" for="menu">
+                        <img class="popup_button-img" src="./popup-icons/popup-button.png" alt="popup button">
+                    </label>
+
+
+                    <div class="popup">
+
+                        <input class="popup_more-input" type="checkbox" name="more" id="more_less">
```

Comment:
When the popup is open, the first interactive element inside the popup must be in focus; in our case, when the user opens the popup and presses the Tab key, the first link should be in focus.